src/Admin/SupportLink.php

```diff
@@ -60,12 +60,9 @@ public function register() {
 	 *
 	 * @return void
 	 */
-	public function admin_bar_menu( $wp_admin_bar ) {
+	public function admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
 
-		if ( empty( $wp_admin_bar ) ||
-			! is_a( $wp_admin_bar, 'WP_Admin_Bar' ) ||
-			! array_key_exists( 'amp', $wp_admin_bar->get_nodes() )
-		) {
+		if ( array_key_exists( 'amp', $wp_admin_bar->get_nodes() ) ) {
```

Comment:
Oops:

```suggestion
		if ( ! array_key_exists( 'amp', $wp_admin_bar->get_nodes() ) ) {
```

Reply:
Also, this can be made more efficient via:

```suggestion
		if ( ! $wp_admin_bar->get_node( 'amp' ) ) {
```